frontend/packages/dev-console/src/components/import/devfile/__tests__/devfile-utils.spec.ts

```diff
@@ -326,7 +326,7 @@ describe('devfile-utils', () => {
     it('should use coFetch for external URIs', async () => {
       const resourceContent = await getResourceContent(
         'outerloop-deploy',
-        'https://raw.githubusercontent.com/jerolimov/devfile-sample-go-basic-remote-k8s-uri/main/outerloop-deploy.yaml',
+        'https://raw.githubusercontent.com/christoph-jerolimov/devfile-sample-go-basic-remote-k8s-uri/main/outerloop-deploy.yaml',
```

Comment:
I am getting 404 for this URL. and old URL https://raw.githubusercontent.com/jerolimov/devfile-sample-go-basic-remote-k8s-uri/main/outerloop-deploy.yaml is working fine.

Reply:
Reverted this change